packages/@aws-cdk/aws-iam/lib/policy-statement.ts

```diff
@@ -7,6 +7,51 @@ import { mergePrincipal } from './util';
  * Represents a statement in an IAM policy document.
  */
 export class PolicyStatement {
+  /**
+   * Creates a new PolicyStatement based on the object provided.
+   * This will accept an object created from the `.toJSON()` call
+   * @param obj the PolicyStatement in object form.
+   */
+  public static fromJson(obj: any) {
+    const ensureArrayOrUndefined = (field: any) => {
```

Comment:
Style: move this out of the method to a free-floating local function. Can this be used `PolicyDocument.fromJson` to validate `Statement`?

Reply:
moved. It's now at the module level.